### ISSUE TITLE
USER STORY: Final Deployment Checks

### ISSUE THREAD

**Issue by rpf13** [rpf13/the-collection#47]
As a **Developer** I can **check my finished project code again, including all deployment settings** so that **I can assure the proper functionality and save deployment on heroku.**

## Acceptance Criteria
- [ ] The deployment steps are checked again one after each other, great attention is set to the **debug** setting in order to not cause any security flaws.
- [ ] The final deployed app is tested on various browsers and devices to make sure that after the last commit, everything works properly 

## Tasks
- [ ] check code again for debug settings and set accordingly
- [ ] check the Heroku vars section, if everything is up to date and correct
- [ ] check the final deployed app

**Comment by rpf13** [comment]
This commit will modify the template structure of the collection app.
It will move the templates for the collection app to a new folder
templates/collection inside the collection app. This is the proper
whay to store the files.
views have been updated with the new path for the templates

During the research for modifying the error page handling, I came
across this rule of thumb to organize it that way - per app.

commit id 830f08a
